### DISCUSSION TITLE
In Dart SDK How to get current login user details?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2069] [Q&A] [answered]
I need to get user details of current login user. How to get the details as well as whether login session is stored in persistant storage?

**Comment by ganigeorgiev** [comment, marked as answer]
You could find some info about the default AuthStore in https://github.com/pocketbase/dart-sdk#authstore.

You can access the authenticated auth model (auth record or admin) via `pb.authStore.model`.

> whether login session is stored in persistant storage?

No. The Dart SDK currently doesn't come with persistent storage since it is intended as a generic Dart package and not just for Flutter. If you are using Flutter and depending on what platform you are targeting you can use localstore, regular or encrypted shared_preferences, key-value store, json files, etc.

You can find 2 example approaches (_custom store and with the `pb.authStore.onChange` stream_) using shared_preferences in https://github.com/pocketbase/pocketbase/discussions/1887#discussioncomment-5057297 (_note that there is also encrypted shared_preferences_).

**Comment by ReyCarrozzakF** [comment]
The approach for getting user details of the current logged in user and checking if the login session is stored in persistent storage depends on the technology stack you are using.
